exonum/src/runtime/rust/mod.rs

```diff
@@ -12,7 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Built-in Rust runtime module.
+//! The basic runtime with the native Rust services.
+//!
+//! This runtime is usually presents in every blockchain instance and can only process a static set
+//! of available Rust services. To change this set you have to recompile the node binary.
+//! During this procedure, be very careful and avoid situations when some of deployed services do
+//! not exist in the list of available. The easiest way to add only a new available artifacts, rather
+//! than delete.
+//!
+//! Rust runtime does not provide any level of service isolation from the operation system, therefore
+//! you should only use a trusted artifacts.
+//!
+//! Artifacts available for deployment are presented by the [`ServiceFactory`][ServiceFactory] objects.
+//!
+//! [ServiceFactory]: trait.ServiceFactory.html
```

Comment:
Not required anymore with [intra-rustdoc links](https://github.com/rust-lang/rust/issues/43466).